classes/help.php

```diff
@@ -155,32 +154,32 @@ function print_page() {
 
 												<tr valign="top">
 													<td width="150">
-														<strong><?php _e( 'Starting Location', 'SimpleMap' ); // XSS ok. ?></strong>
+														<strong><?php _e( 'Starting Location', 'simplemap' ); // XSS ok. ?></strong>
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 132 characters

<!-- Comment by [SideCI](https://sideci.com) -->